packages/server/modules/activitystream/tests/activitySummary.spec.ts

```diff
@@ -31,6 +51,32 @@ const createActivitySummary = createActivitySummaryFactory({
   getStream,
   getActivity: getActivityFactory({ db })
 })
+const createStream = legacyCreateStreamFactory({
```

Comment:
most of the added lines from having to copy paste this factory invocation over and over again....